.bazelrc

```diff
@@ -61,9 +61,10 @@ build:asan --features=asan
 
 # Configuration for enabling LibFuzzer (along with ASan).
 build:fuzzer --features=fuzzer
-
-# Proto fuzzer specific configuration.
-build:proto-fuzzer --features=proto-fuzzer
+build:fuzzer --features=-libc++
+build:fuzzer --@rules_fuzzing//fuzzing:cc_engine=@rules_fuzzing//fuzzing/engines:libfuzzer
+build:fuzzer --@rules_fuzzing//fuzzing:cc_engine_instrumentation=libfuzzer
+build:fuzzer --@rules_fuzzing//fuzzing:cc_engine_sanitizer=asan
```

Comment:
Not blocking, just so we don't forget, but if all this works we may want to come back and add UBSan stuff somewhere.